ui/components/multichain/pages/send/components/content.tsx

```diff
@@ -1,4 +1,20 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import { Box } from '../../../../component-library';
+import { getSendHexDataFeatureFlagState } from '../../../../../ducks/metamask/metamask';
+import { getSendAsset } from '../../../../../ducks/send';
+import { AssetType } from '../../../../../../shared/constants/transaction';
+import { SendHexData } from '.';
 
-export const SendPageContent = () => <Box>(Content)</Box>;
+export const SendPageContent = () => {
+  const showHexDataFlag = useSelector(getSendHexDataFeatureFlagState);
+  const asset = useSelector(getSendAsset);
+
+  const showHexData =
+    showHexDataFlag &&
+    asset &&
+    asset.type !== AssetType.token &&
+    asset.type !== AssetType.NFT;
+
+  return <Box>{showHexData ? <SendHexData /> : null}</Box>;
```

Comment:
Can we add a className or data-testId to this component since this component doesn't have a story and will be helpful?

Reply:
We can add those when we add some basic jest tests in the future.